src/main/kotlin/kmu/KMU.kt

```diff
@@ -3,6 +3,7 @@ package kmu
 
 inline fun <T> T.match(predicate: (T) -> Boolean) = predicate(this)
 
-inline fun <T> XXX(): T = throw NotImplementedError()
+@Suppress("FunctionName")
+fun <T> XXX(): T = throw NotImplementedError()
```

Comment:
why did you remove the inline?

Reply:
I was guided by this:
* compiler warning: `Expected performance impact of inlining 'public inline fun <T> XXX(): T defined in kmu in file KMU.kt' can be insignificant. Inlining works best for functions with lambda parameters`
* [When we don’t have function type parameter, reified type parameter, and we don’t need non-local return, then we most likely shouldn’t use inline modifier.](https://blog.kotlin-academy.com/effective-kotlin-consider-inline-modifier-for-higher-order-functions-758afcaffc11) 
* [KT-21020](https://youtrack.jetbrains.com/issue/KT-21020)

But, having looked at how the [TODO()](https://github.com/JetBrains/kotlin/blob/master/libraries/stdlib/src/kotlin/util/Standard.kt#L22) method was declared in Kotlin, I think that, most likely, I was wrong with XXX :(
return inline?